variables.tf

```
@@ -27,20 +27,44 @@ variable "lacework_image" {
   default     = "lacework/datacollector"
 }
 
+variable "lacework_image_pull_policy" {
+  type        = string
+  description = "The pull policy to use for deploying the Lacework datacollector"
+  default     = "IfNotPresent"
+}
+
 variable "namespace" {
   type        = string
   description = "The Kubernetes namespace in which to deploy"
   default     = "default"
 }
 
-variable "pod_cpu" {
+variable "pod_cpu_request" {
   type        = string
   default     = "100m"
-  description = "The amount of CPU units to assign to the Lacework datacollector pod"
+  description = "The amount of CPU units to request for the Lacework datacollector pod"
 }
 
-variable "pod_mem" {
+variable "pod_mem_request" {
   type        = string
   default     = "256Mi"
```

I think the default memory is `512Mi`
```suggestion
  default     = "512Mi"
```